mobile/Yote/js/AuthNavigator.js

```diff
@@ -18,7 +18,7 @@ import UserRegister from '../js/resources/user/views/UserRegister';
 
 export default function AuthNavigator() {
   // use the hook to get the loggedInUser from the authStore
-  const { loggedInUser } = useLoggedInUser();
+  const token = useSessionToken();
```

Comment:
maybe not for future, but technically just having "token" doesn;t meant they're logged in, since the token could have been logged out from the server. technically you *should* use logged in user, it just needs to be fetched FROM the token somehow first

Reply:
ummmm okay i can look into that a bit. there are a few ways we could go about it i think